charts/open-notificaties/README.md

```diff
@@ -34,7 +44,7 @@ helm install open-notificaties open-zaak/open-notificaties \
 | `tags.redis` | Install Redis subchart | `true` |
 | `tags.rabbitmq` | Install RabbitMQ subchart | `true` |
 | `image.repository` | The repository of the Docker image | `openzaak/open-notificaties` |
-| `image.tag` | The tag of the Docker image | `latest` |
+| `image.tag` | The tag of the Docker image | `1.2.1` |
```

Comment:
Would be nice to update `image.tag` in values.yaml as well.

Reply:
We could also start using appVersion, and do not set image.tag by default. See: https://github.com/open-zaak/charts/blob/main/charts/open-zaak/templates/deployment.yaml#L36

Reply:
not a bad shout - I also see that `appVersion` needs to be bumped still

Reply:
Yes 😀